go-apps/meep-metrics-engine/server/metrics-engine.go

```diff
@@ -19,15 +19,67 @@ package server
 import (
```

Comment:
Separate code into version-specific folders.
Remove version references.
Adapt code to object name changes.

Reply:
fixed